docs/source/components/nodes/tof.rst

```diff
@@ -30,20 +30,94 @@ Inputs and Outputs
 .. code-block::
 
               ┌───────────┐    depth
-  inputConfig |           ├────────►
-  ───────────►│           | amplitude
-     input    |    ToF    ├────────►
-  ───────────►│           │    error
+              │           ├────────►
+  inputConfig |           | amplitude
+  ───────────►│           ├────────►
+              │    ToF    │    phase
+     input    |           ├────────►
+  ───────────►│           │ intensity
               │           ├────────►
               └───────────┘
 
+
 **Message types**
 
 - ``inputConfig`` - :ref:`ToFConfig`
 - ``input`` - :ref:`ImgFrame`
-- ``depth`` - :ref:`ImgFrame`
+- ``depth`` - :ref:`ImgFrame` - Decoded depth map
 - ``amplitude`` - :ref:`ImgFrame`
-- ``error`` - :ref:`ImgFrame`
+- ``phase`` - :ref:`ImgFrame` Phase image, useful for debugging (FP32)
+- ``intensity`` - :ref:`ImgFrame`
+
+ToF Settings
+############
+
+In :ref:`ToF depth` example we allow users to quickly configure ToF settings.
+
+Here are the most important settings:
+
+* Optical Correction: It's a process that corrects the optical effect. When enabled, the ToF returns depth map (represented by Green Line on graph below) instead of distance, so it matches :ref:`StereoDepth` depth reporting. It does rectification and distance to depth conversion (Z-map).
+* Phase Unwrapping - Process that corrects the phase wrapping effect of the ToF sensor. You can set it to [0..5 are optimized]. The higher the number, the longer the ToF range, but it also increases the noise. Approximate max distance (for exact value, see :ref:`Max distance` below):
+
+  * `0` - Disabled, up to ~1.87 meters (utilizing 80MHz modulation frequency)
+  * `1` - Up to ~3 meters
+  * `2` - Up to ~4.5 meters
+  * `3` - Up to ~6 meters
+  * `4` - Up to ~7.5 meters
+
+* Burst mode: When enabled, ToF node won't reuse frames, as shown on the graph below. It's related to post-processing of the ToF frames, not the actual sensor/projector. It's disabled by default.
+* Phase shuffle Temporal filter: Averages shuffled and non-shuffled frames of the same modulation frequency to reduce noise. It's enabled by default. You can disable it to reduce :ref:`ToF motion blur` and system load.
+
+These are mostly for debugging, and should be enabled:
```

Comment:
Not sure, should we document these given that users should not play with these? It's mostly for us